cmake/edge_include_directories.cmake

```
@@ -45,12 +45,11 @@ include_directories (${MBED_CLOUD_CLIENT_DEPENDENCY_SOURCES}/mbed-client/mbed-cl
 
 MESSAGE ("Is Subdevice FOTA enable - ${ENABLE_SUBDEVICE_FOTA}")
 # update client
-if (ENABLE_SUBDEVICE_FOTA)
+if (NOT ENABLE_SUBDEVICE_FOTA)
```

Can we change this to 
```suggestion
if (ENABLE_UC_HUB)
```

---

Implemented